meta-ibm/recipes-phosphor/webui/webui-vue/changeLogo.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+model=($(busctl get-property xyz.openbmc_project.Inventory.Manager /xyz/openbmc_project/inventory/system/chassis/motherboard/bmc xyz.openbmc_project.Inventory.Decorator.Asset SubModel))
+DIR=/usr/share/www/img
+if [ "${model}" = '"S0"' || "${model}" = '"D0"' ]; then
+    echo "loading IBM logo"
```

Comment:
Lets just say "Using IBM logo" and then remove the next 2 lines. There is no need to do the mounts because the default is the IBM logo.

Reply:
+1

Reply:
Done